features/support/env.rb

```diff
@@ -1,5 +1,35 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__) + '/../../lib')
-require 'aruba/cucumber'
+
+# Has to be the first file required so that all other files show coverage information
+require 'simplecov'
```

Comment:
Is there a better place to put `simplecov_setup` so it's not loaded by the cucumber process, but available for require by child processes?  Should I make a different directory than `features/support`?